docs/guides/deploy.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Deploying your first project with Nitric Deploy
+description: Nitric Deploy builds and deploys the right infrastructure from your code in your chosen cloud.
+---
+
+## What we'll be doing
+
+1. Sign to Nitric Deploy
+2. Create an organization
+3. Set up AWS credentials
+4. Create or import a project
+5. Create an environment
+6. Configure environment variables
+7. Deploy your project
+8. Check out the logs
+
+<br/>
+
+## Sign to Nitric Deploy
+
+> Nitric Deploy is currently in Alpha, you'll need to [request access](https://deploy.nitric.io/signup).
+
+> Your email address will need to match your github account.
```

Comment:
Instead of making this a second alert, could you just add it after an "and" in the sentence above?

Reply:
Actually, would suggest altering it as I suggested in the previous comment.